lib/chef/data_collector/run_end_message.rb

```diff
@@ -16,6 +16,7 @@
 #
 
 require_relative "message_helpers"
+require_relative "../dist"
```

Comment:
?

Reply:
I don't see an issue, but then again I'm noobsauce o.O What's wrong with this?

Reply:
nm, I see it now XD